net-im/telegram-desktop/telegram-desktop-3.5.2-r1.ebuild

```diff
@@ -95,17 +97,20 @@ src_prepare() {
 }
 
 src_configure() {
-	# gtk is really needed for image copy-paste due to https://bugreports.qt.io/browse/QTBUG-56595
 	local mycmakeargs=(
 		-DTDESKTOP_LAUNCHER_BASENAME="${PN}"
 		-DCMAKE_DISABLE_FIND_PACKAGE_tl-expected=ON  # header only lib, some git version. prevents warnings.
 		-DDESKTOP_APP_QT6=OFF
 
+		-DDESKTOP_APP_DISABLE_DBUS_INTEGRATION=$(usex dbus no yes)
 		-DDESKTOP_APP_DISABLE_X11_INTEGRATION=$(usex X no yes)
 		-DDESKTOP_APP_DISABLE_WAYLAND_INTEGRATION=$(usex wayland no yes)
-		-DDESKTOP_APP_DISABLE_DBUS_INTEGRATION=$(usex dbus no yes)
 		-DDESKTOP_APP_DISABLE_SPELLCHECK=$(usex spell no yes)  # enables hunspell (recommended)
 		-DDESKTOP_APP_USE_ENCHANT=$(usex enchant)  # enables enchant and disables hunspell
+
+		# This option is heavily discouraged by upstream.
+		# See files/tdesktop-*-jemalloc-optional.patch
+		-DDESKTOP_APP_DISABLE_JEMALLOC=$(usex jemalloc no yes)
```

Comment:
Consider:
```suggestion
		-DDESKTOP_APP_DISABLE_JEMALLOC=$(usex !jemalloc)
```

Reply:
Ah, I wasn't aware it accepted this syntax. Thanks!